api/infrastructure/parallelcluster-api.yaml

```diff
@@ -302,8 +316,9 @@ Resources:
             Principal:
               AWS:
                 - !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:root
+      PermissionsBoundary: !If [HasPermissionBoundary, !Ref PermissionsBoundaryPolicy, !Ref 'AWS::NoValue']
       Policies:
-        - PolicyName: ParallelClusterApiInvokePolicy
+        - PolicyName: !Sub ${IAMRoleAndPolicyPrefix}ParallelClusterApiInvokePolicy
```

Comment:
Why `StackIdSuffix` is not applied to PolicyName?

Reply:
Policies have stack-id added to them, which is an automatic disambigator